test/parallel/test-net-connect-buffer.js

```diff
@@ -69,12 +69,12 @@ tcp.listen(0, common.mustCall(function() {
     [],
     {}
   ].forEach((value) => {
-    common.expectsError(() => socket.write(value), {
+    socket.write(value, common.expectsError({
```

Comment:
Can you add a comment to explain this? It might be obvious but I had to spend some time to understand. The `'error'` event is only emitted once and it is emitted for the write on line 55, so now the callback is used.

Can you also please fix comment on line 54?